ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -503,9 +503,20 @@ objects:
 
         if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
           workers=3
-          if [[ "${CLUSTER_VARIANT}" == "compact" ]]; then
+          if [[ "${CLUSTER_VARIANT}" =~ "compact" ]]; then
```

Comment:
@smarterclayton hopefully this is okay?

Reply:
I really want whatever new hotness the test-platform folks are cooking up to allow us to mix in assorted flavors in ci-operator.  Until then, I think regexp/substring matching like this is fine.